test-utils/src/dev/falcon.rs

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Mechanisms to launch and control propolis VMs via falcon
+
+#[cfg(test)]
+mod test {
```

Comment:
Idea, take it or leave it -- it might be kinda nice to have a way to validate "this test is being run from a VMM context, bail out if that isn't true?"

I dunno what we'd want to set in the `falcon_runner`, but just *something* to distinguish it from the host, since "getting the config right" is one of the trickiest bits to launch these VM tests right now

Reply:
I added a check for the hostname, which is quite unique. We can certainly inject a file though, which might work better once we start using different names for VMs. We'll want to use different names to run tests in parallel.